docs/user-guide/replicasets.md

```diff
@@ -0,0 +1,95 @@
+
+Replica set is the next generation replication controller. The only difference between a _replica set_ and a [_replication controller_](replication-controller.md) right now is the selector support. Replica set supports the new set-based selector requirements as described in the [labels user guide](labels.md#label-selectors) whereas a replication controller only supports equality-based selector requirements.
+
+While replica sets can be used indepently, today it's mainly used by [deployments](deployments.md) as a mechanism to orchestrate pod creation, deletion and updates. When you use deployments you don't have to worry about manging the replica sets that they create. Deployments own and manage their replica sets.
+
+### Example
+
+{% include code.html language="yaml" file="replicasets/frontend.yaml" ghlink="/docs/user-guide/replicasets/frontend.yaml" %}
+
+Saving this config into `frontend.yaml` and submitting it to a Kubernetes cluster should create the defined replica set and the pods that it manages.
+
+```sh
+$ kubectl create -f frontend.yaml 
+replicaset "frontend" created
+$ kubectl describe rs/frontend
+Name:		frontend
+Namespace:	default
+Image(s):	gcr.io/google_samples/gb-frontend:v3
+Selector:	tier=frontend,tier in (frontend)
+Labels:		app=guestbook,tier=frontend
+Replicas:	3 current / 3 desired
+Pods Status:	3 Running / 0 Waiting / 0 Succeeded / 0 Failed
+No volumes.
+Events:
+  FirstSeen	LastSeen	Count	From				SubobjectPath	Type		Reason			Message
+  ---------	--------	-----	----				-------------	--------	------			-------
+  1m		1m		1	{replicaset-controller }			Normal		SuccessfulCreate	Created pod: frontend-qhloh
+  1m		1m		1	{replicaset-controller }			Normal		SuccessfulCreate	Created pod: frontend-dnjpy
+  1m		1m		1	{replicaset-controller }			Normal		SuccessfulCreate	Created pod: frontend-9si5l
+$ kubectl get pods
+NAME             READY     STATUS    RESTARTS   AGE
+frontend-9si5l   1/1       Running   0          1m
+frontend-dnjpy   1/1       Running   0          1m
+frontend-qhloh   1/1       Running   0          1m
+```
+
+## Replica set as an Horizontal Pod Autoscaler target
+
+A replica set can also be a target for [Horizontal Pod Autoscalers (HPA)](horizontal-pod-autoscaler.md). Here is an example HPA targeting the replica set we created in the previous example.
+
+```yaml
+apiVersion: autoscaling/v1
+kind: HorizontalPodAutoscaler
+metadata:
+  name: frontend-scaler
+spec:
+  scaleRef:
+    kind: ReplicaSet
+    name: frontend
+    subresource: scale
+  minReplicas: 3
+  maxReplicas: 10
+  cpuUtilization:
+    targetPercentage: 50
+```
+
+## When to use a replica set?
+
+A replica set ensures that a specified number of pod “replicas” are running at any one time. However, a deployment is a higher level concept that manages replica sets and provides declarative updates to pods along with a lot of other useful features. Therefore, we recommend using deployments instead of directly using replica sets, where possible.
```

Comment:
higher-level

s/where possible/unless you require custom update orchestration or don't require updates at all/